datadog_checks_base/datadog_checks/base/stubs/aggregator.py

```diff
@@ -23,6 +27,15 @@ def normalize_tags(tags, sort=False):
     return tags
 
 
+def backend_normalize_metric_name(metric_name):
```

Comment:
I think we should make https://github.com/DataDog/integrations-core/blob/a3de7a27db9e29e138ba3ab9e27f3d0b0f45afc8/datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/metadata.py#L201-L211 reuse `backend_normalize_metric_name` to avoid duplication.

To make it more natural to import that in `validate/metadata.py` we might want to move `backend_normalize_metric_name` to a utils.py file. Any suggestion where ? cc @ofek

Reply:
There's not really a good way, just duplicate

Reply:
We already depend on base https://github.com/DataDog/integrations-core/blob/ea9f97c8950f947e215897ed257fff8a43707497/datadog_checks_dev/datadog_checks/dev/plugin/pytest.py#L41-L44

Why not make `validate/metadata.py` import `backend_normalize_metric_name` ?

Reply:
We do not depend on base. It's guaranteed in tox, not as ddev cli

Reply:
👍  I see.